src/include/duckdb/common/types/value.hpp

```diff
@@ -139,6 +143,9 @@ class Value {
 		return Value(*this);
 	}
 
+	//! Convert this value to a string as is
+	DUCKDB_API string ToStringAsIs() const;
```

Comment:
Agreed with Richard, perhaps we can call this `GetBlob` to be consistent with the `GetValue` syntax, and our definition of the type as blob?

Reply:
I reverted this change.